### PR TITLE
Added FaceLiveness Code infra

### PR DESCRIPTION
**Pull request by vineetkachhawaha**
*Description of changes:*
* Added AWS CDK for Amazon Cognito for FaceLiveness
* Added AWS CDK code for FaceLiveness Lambda 
* Bug fixes - CompareFace , IndexFaces

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: infra/configsettings.py [hunk 1]
```diff
@@ -89,6 +89,18 @@ def use_custom_asset_bucket(self)->bool:
...
     return self.json['use_custom_asset_bucket']
+  
+  @property
+  def use_inventory_bucket(self)->bool:
+    if 'use_inventory_bucket' in self.json:
+      return self.json['use_inventory_bucket']
+    return False
+  
+  @property
+  def use_debug_statet(self)->bool:
```
Comment (dr-natetorious) [inline infra/configsettings.py:100]: What's a `statet`? Do you mean `use_debug_state_machine`

File: infra/default_lambda.py [hunk 2]
```diff
@@ -91,8 +98,14 @@ def __init__(self, scope: Construct, id: str, riv_stack:IVpcRivStack,subnet_grou
     '''
     Include the shared requirements.txt 
     '''
+    if not os.environ.get("SKIP_PIP"):
+      if not os.path.isdir(output_dir): 
+        subprocess.check_call(
```
Comment (dr-natetorious) [inline infra/default_lambda.py:103]: This command will break on cross-platform (e.g., Windows calls the command `pip3`)

File: infra/frontend/topology.py [hunk 3]
```diff
@@ -55,6 +56,14 @@ def __init__(self, scope: Construct, id: builtins.str, riv_stack: IVpcRivStack,
                                     )
         self.amplify.add_environment(
             name="REACT_APP_ENV_API_URL", value=apigateway.rest_api_url())
+        self.amplify.add_environment(
```
Comment (dr-natetorious) [inline infra/frontend/topology.py:59]: Do we also need a `config.use_cognito` flag?

File: infra/configsettings.py (hunk 1 above)
Comment (vineetkachhawaha) [reply infra/configsettings.py]: Fixed the typo issue

File: infra/frontend/topology.py (hunk 3 above)
Comment (vineetkachhawaha) [reply infra/frontend/topology.py:59]: Nope because for FE Liveness , Cognito is prerequisite.

File: infra/default_lambda.py (hunk 2 above)
Comment (vineetkachhawaha) [reply infra/default_lambda.py:103]: i change it to pip3

**Pull request merged**